moto/awslambda/responses.py

```diff
@@ -300,6 +302,15 @@ def _delete_function(self, request, full_url, headers):
         else:
             return 404, {}, "{}"
 
+    @staticmethod
+    def _update_function_configuration(configuration, qualifier):
+        configuration = {k: v for k, v in configuration.items()}
```

Comment:
Do we need the copy? As far as I can tell, the entire `configuration` object is created from scratch when calling `fn.get_code()`

Reply:
:+1: